### ISSUE TITLE
Segmentation Fault 11 incrementing Float Value

### ISSUE THREAD

**Issue by cpats007** [aerospike/aerospike-client-php#109]
Hi. When trying to increment a float value, I get a Segmentation fault: 11 and it fails.

Test code I'm using to recreate the issue:

```php
        $uid = uniqid('inc_');
        $key = $this->getDb()->initKey(self::KEY_DB_NAME, 'xc_bids', $uid);

        // store the key data with the record, rather than just its digest
        $options = [
            Aerospike::OPT_POLICY_KEY    => Aerospike::POLICY_KEY_SEND,
        ];

        $floatVal = 1.234;

        $status = $this->getDb()->increment($key, 'float', $floatVal, $options);
        echo $status . PHP_EOL;

        // Check the outcome of the adding:
        if ($status !== Aerospike::OK) {
            $this->echoToConsole('Summink Went Wrong');
        } else {
            return true;
        }

        return false;
```

Aerospike Client version: 3.4.9
Aerospike Server Version: 3.9.0.3

PHP Info:

PHP 5.6.22 (cli) (built: Jul 19 2016 13:23:48) 
Copyright (c) 1997-2016 The PHP Group
Zend Engine v2.6.0, Copyright (c) 1998-2016 Zend Technologies
    with Xdebug v2.4.0, Copyright (c) 2002-2016, by Derick Rethans

**Comment by aminer** [comment]
Thanks for reporting this! Looking into it now.

**Comment by aminer** [comment]
This is issue CLIENT-785 internally. 
The increment value can be integer, float, or string (if numeric). The fix will be included in the next release.

**Comment by cpats007** [comment]
No problem, that's a great fix. Do you have an ETA on when this will be released? It's something we didn't envisage not working and so we're kind of desperate for it heh. Thanks

**Comment by aminer** [comment]
The next client release is scheduled for next week 😉

**Comment by aminer** [comment]
Fixed in release 3.4.10.

**Comment by jboone100** [comment]
Closing this. Please reopen if still a problem.
